doc/specs/#166 - Font Support.md

```diff
@@ -0,0 +1,416 @@
+---
+author: Ryan Fu @ryfu-msft
+created on: 2024-10-1
+last updated: 2024-10-1
+issue id: 166
+---
+
+# Support for installation of fonts
+
+For [#166](https://github.com/microsoft/winget-cli/issues/166)
+
+## Abstract
+This spec outlines the design for supporting the installation of fonts.
+
+## Context
+
+### Font File Types
+
+This feature will support font files with the following file types:
+- .ttf (True Type File) - Most common format
+- .ttc (True Type Collection)
+- .otf (Open Type Font)
+
+The following font types to my knowledge are not supported:
+- .woff & .woff2 (developed by Google for the modern browser)
+- .eot (Embedded OpenType font file) Not used because of security issues.
+
+> Font types will not be treated differently. Support for font installation will be determined by calling [IDWriteFontFile::Analyze](https://learn.microsoft.com/en-us/windows/win32/api/dwrite/nf-dwrite-idwritefontfile-analyze).
+
+### Installing Fonts
+Typically, a user would download the font file and drag it to the `C:\Windows\Fonts` directory in the explorer view, but there are a couple things happening behind the scenes.
```

Comment:
Does the drag and drop flow trigger the same endpoints as double clicking the file in File Explorer? Just trying to understand if all the current methods of installing fonts end up at the same place